docs/LV_client/tutorials/Simulator_configuration.md

```diff
@@ -0,0 +1,9 @@
+## Simulator Configuration
+
+This tutorial demonstrates how to use the LabVIEW client to configure the behavior of vehicles in the scene as well as the object's materials when using the monoDrive Simulator. 
```

Comment:
This tutorial demonstrates how to configure the behavior of vehicles and an object's materials when using the monoDrive Simulator.